File: SantanderTeste/app/src/main/java/com/example/jcsantos/santanderteste/Components/Objects/Statement.java
package com.example.jcsantos.santanderteste.Components.Objects;

import java.io.Serializable;

public class Statement implements Serializable {
    String title;
    String desc;
    String date;
    String value;

    public String getTitle() {

        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return desc;
    }

    public void setDescription(String description) {
        this.desc = description;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: app/src/main/java/com/solinftec/desafiosantander_rafaelpimenta/model/Statement.kt
package com.solinftec.desafiosantander_rafaelpimenta.model

import java.io.Serializable

data class Statement(
    val title: String,
    val desc: String,
    val date: String,
    val valueFormatted: String,
    val value: Double
) : Serializable
